src/main/java/com/rultor/agents/github/CommentsTag.java

```diff
@@ -71,6 +77,12 @@
         ResourceBundle.getBundle("phrases");
 
     /**
+     * Version pattern.
+     */
+    private static final Pattern VERSION_PATTERN =
+        Pattern.compile("\\.?(?:\\d+\\.)*\\d+");
```

Comment:
@gumbelmj This matches for example ".1", this is not sufficient for validating a version.